storagenode/operator/operatorserver/server.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package operatorserver
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"strings"
+
+	"github.com/prometheus/common/log"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+
+	"storj.io/storj/storagenode/operator"
+)
+
+// TODO: improve embedded resources generation
+//go-bindata -pkg operatorserver -o storagenode/operator/operatorserver/static.go web/operator/dist web/operator/dist/public/
+
+// Error is storagenode operator web error type
+var Error = errs.Class("storagenode operator web error")
+
+// Config contains configuration for storagenode operator web server
+type Config struct {
+	Address   string `help:"server address of the graphql api gateway and frontend app" default:"127.0.0.1:14002"`
+	StaticDir string `help:"path to static resources" default:""`
```

Comment:
I think we should set the default to `web/operator` here, to avoid problems in the static.go file